packages/truffle/test/scenarios/commands/help.js

```diff
@@ -31,6 +32,7 @@ describe("truffle help [ @standalone ]", function () {
     }).timeout(20000);
 
     it("displays help for the given command with help function [ @>=12 ]", async function () {
```

Comment:
Interesting! We will have to revisit the purpose of this `[ @>=12 ]` tag when we retire node 12